packages/trader/src/exchange-adapters/CreateExchangeAdapter.ts

```diff
@@ -13,7 +14,7 @@ async function createExchangeAdapter(
 ) {
   assert(config.type, "Exchange adapter must have a type. EG uniswap for a uniswap dex");
 
-  if (config.type === "uniswap") {
+  if (config.type === "uniswap-v2") {
```

Comment:
this is a breaking change! we must update the current trader at the same time that this is rolled out.